src/components/playback/playerSelectionMenu.js

```diff
@@ -141,18 +141,18 @@ export function show(button) {
 
 function showActivePlayerMenu(playerInfo) {
     Promise.all([
-        import('dialogHelper'),
-        import('dialog'),
-        import('emby-checkbox'),
-        import('emby-button')
+        import('../dialogHelper/dialogHelper'),
+        import('../dialog/dialog'),
+        import('../../elements/emby-checkbox/emby-checkbox'),
+        import('../../elements/emby-button/emby-button')
     ]).then(([dialogHelper]) => {
         showActivePlayerMenuInternal(dialogHelper, playerInfo);
     });
 }
 
 function disconnectFromPlayer(currentDeviceName) {
     if (playbackManager.getSupportedCommands().indexOf('EndSession') !== -1) {
-        import('dialog').then(({default: dialog}) => {
+        import('../dialog/dialog').then(({default: dialog}) => {
```

Comment:
_[not related to PR]_ how can `dialog` be a function?
https://github.com/jellyfin/jellyfin-web/blob/2c899c93dff42d7cefc1c2cbeb0e63eaf5d02b8b/src/components/playback/playerSelectionMenu.js#L167

Reply:
fixed as suggested